pages/dug_12/4-shameless-plug.md

```diff
@@ -0,0 +1,619 @@
+  <img src="/@fs/repo/img/dug_11/dpp2.png" width="220" style="margin-left: 50px"/>
+  <img src="/@fs/repo/img/dug_11/dpp3.png" width="220" style="margin-left: 50px"/>
+</div>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/dasharo-pro-package](https://shop.3mdeb.com/product-category/dasharo-pro-package)
+</center>
+
+---
+
+<br>
+
+<div style="display: flex; justify-content: center; align-items: center;">
+  <img src="/@fs/repo/img/dug_11/dpp4.png" width="220"/>
+  <img src="/@fs/repo/img/dug_11/dpp5.png" width="220" style="margin-left: 50px"/>
+</div>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/dasharo-pro-package](https://shop.3mdeb.com/product-category/dasharo-pro-package)
+</center>
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+---
+
+# Accessories
+
+---
+
+<center><img src="/@fs/repo/img/dug_11/privacy-screen.png" width="280"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/accessories/](https://shop.3mdeb.com/product-category/accessories/)
+</center>
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+# https://shop.3mdeb.com
+
+<!--
+
+Everything else you will buy on 3mdeb.com
+
+-->
+---
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+# Backlog
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+# Services
+
+---
+
+## <center>Dasharo Services</center>
+
+<br>
+
+<center><img src="/@fs/repo/img/dug_12/dasharo_services.png" width="900"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/dasharo-services/](https://shop.3mdeb.com/product-category/dasharo-services/)
+</center>
+
+---
+
+## <center>Zarhus Services</center>
+
+<br>
+
+<center><img src="/@fs/repo/img/dug_8/zarhus_services.png" width="900"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/shop/zarhus-services/](https://shop.3mdeb.com/shop/zarhus-services/)
+</center>
+
+---
+
+## <center>Services</center>
+
+<br>
+
+<center><img src="/@fs/repo/img/dug_12/services1.png" width="900"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/services/](https://shop.3mdeb.com/product-category/services/)
+</center>
+
+---
+
+## <center>Services</center>
+
+<br>
+
+<center><img src="/@fs/repo/img/dug_12/services2.png" width="900"></center>
+
+<br>
+
+<center>
+
+### [https://shop.3mdeb.com/product-category/services/](https://shop.3mdeb.com/product-category/services/)
+</center>
+
+---
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+# Highlighting HCL Reports
+
+---
+
+<center><img src="/@fs/repo/img/dug_12/hcl_highlights.png" width="500"></center>
+
+<br>
+
+<center>
+
+### [https://docs.dasharo.com/unified/msi/hcl/](https://docs.dasharo.com/unified/msi/hcl/)
+</center>
+
+<!--
+
+Add: Here are the configurations confirmed compatible, the newly verified ones, and those you can already purchase from us (listed on each product’s page).
```

Comment:
@Stanislaw-bnk but the page shows old dates. I understand that these are the dates of the last update.
It would be useful to add a comment explaining why, despite uploading the HCL to the database, we are not extracting information from it and posting it on the website. I mean, an explanation of why old dates are shown there.

<img width="860" height="686" alt="image" src="https://github.com/user-attachments/assets/03a6fe5e-9177-456e-bdb3-fccf7743c8df" />